cli/commands.go

```diff
@@ -77,5 +77,11 @@ var (
 				flKey,
 			},
 		},
+		{
+			Name:    "export",
```

Comment:
There is a `print` subcommand in the tool, which prints envs in various formats. This functionality should be a new print format instead of a new subcommand.

Reply:
Here is the active list of the output formats.

And this is how these can be used:
<img width="470" alt="Screen Shot 2020-07-13 at 9 25 37" src="https://user-images.githubusercontent.com/1564933/87306173-eaa7b300-c517-11ea-9747-3f4aa4794fd7.png">

Also please make sure that this will print an evaluable format so escaping strings and so on.
For example:
```
ENVKEY="My custom text including space, =, \"."
```

Reply:
please give the name "envlist" to the new printing format.